admin_guide/manage_authorization_policy.adoc

```diff
@@ -565,3 +565,81 @@ administrator, you can grant them the abilities.
 ----
 $ oadm policy add-role-to-user system:daemonset-admin <user>
 ----
+
+[[manage-authorization-policy-creating-local-role]]
+== Creating a Local Role
+
+To create a local role for a project, you can either copy and modify an existing
+role or build a new role from scratch. It is recommended that you build it from
+scratch so that you understand each of the permissions assigned.
+
+To copy the cluster role *view* to use as a local role, run:
+
+----
+$ oc get clusterole view -o yaml > clusterrole_view.yaml
+$ cp clusterole_view.yaml localrole_exampleview.yaml
+$ vim localrole_exampleview.yaml
+# 1. Update kind: ClusterRole to kind: Role
+# 2. Update name: view to name: exampleview
+# 3. Remove resourceVersion, selfLink, uid, and creationTimestamp
+$ oc project <project_you_want_to_add_the_local_role_exampleview_to>
```

Comment:
this could be more or less confused to user, could combine this comand together with next line `oc create -f path/to/localrole_exampleview.yaml -n <project_you_want_to_add_the_local_role_exampleview_to>` .